src/labels/CustomLabels.labels

```diff
@@ -7264,6 +7264,78 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Title of checklist item Train Users in Get Started</shortDescription>
         <value>Train Your users</value>
     </labels>
+    <labels>
+        <fullName>gsChecklistItemWhatIsNPCloudLink</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link text for What is Nonprofit Cloud checklist item in Welcome section</shortDescription>
+        <value>Learn more about the Nonprofit Cloud in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemWhatIsNPCloudTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title for checklist item What is Nonprofit Cloud in Welcome section</shortDescription>
+        <value>What is Nonprofit Cloud?</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemWhatIsNPCloudTitleDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for checklist item What is Nonprofit Cloud in Welcome section</shortDescription>
+        <value>Nonprofit Cloud includes apps, tools, features, services, and technology designed to meet the unique needs of nonprofits.</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemWhatIsNPSPDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description for checklist item What is NPSP in Welcome section</shortDescription>
+        <value>The foundation of Nonprofit Cloud is the Nonprofit Success Pack (NPSP) app. NPSP helps you manage your constituents and fundraising efforts. If someone says they’re using the “nonprofit version” of Salesforce, they’re usually talking about NPSP.</value>
```

Comment:
Single quotes need to be escaped as `&apos;` and double quotes need to be escaped as `&quot;`. There are other new labels in this PR that need the same fix.
```suggestion
        <value>The foundation of Nonprofit Cloud is the Nonprofit Success Pack (NPSP) app. NPSP helps you manage your constituents and fundraising efforts. If someone says they&apos;re using the &quot;nonprofit version&quot; of Salesforce, they&apos;re usually talking about NPSP.</value>
```